pkg/test/resourcefixture/testdata/basic/bigqueryconnection/v1alpha1/bigqueryconnectionconnection/_generated_object_bigqueryconnectionconnection.golden.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: bigqueryconnection.cnrm.cloud.google.com/v1alpha1
+kind: BigQueryConnectionConnection
+metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/observed-secret-versions: (removed)
+    cnrm.cloud.google.com/state-into-spec: absent
```

Comment:
Can you remove the state-into-spec annotation? We are deprecating this annotation completely?

Reply:
sounds good, thanks!